README.md

```diff
@@ -7,8 +7,11 @@ curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh | b
 2. Restart your bash session and install node using NVM: `nvm install node`
 3. In this directory, install the node packages, including webpack: `npm install`
 
+## Testing the App
+- Run `npm test`. This will lint your test files, and then run them.
+
 ## Running the App
-- Run `npm run build`.  This will start webpack, which will "watch" the files in `src/` and will update the bundle automatically.  This will also run eslint whenever your code changes so you can see if the linter fails.
+- Run `npm run build`.  This will start webpack, which will "watch" the files in `src/` and will update the bundle automatically.  This will also run eslint whenever your code changes so you can see if the linter fails. It will also automatically run unit tests whenever you save a file it is watching.
```

Comment:
![screenshot 2017-07-13 06 40 34](https://user-images.githubusercontent.com/9058492/28162779-444b3752-6796-11e7-8855-021b073b7e68.png)

Should I do something besides just running `npm run build`?